ffwd-http-reporter/src/main/java/com/spotify/metrics/ffwdhttp/FastForwardHttpReporter.java

```diff
@@ -427,17 +451,20 @@ public void close() {
         private final long timestamp;
         private final String key;
         private final Map<String, String> tags;
+        private final Map<String, String> resources;
         private final String unit;
         private final String metricType;
 
         public BatchBuilder(
             final List<Batch.Point> points, final long timestamp, final String key,
-            final Map<String, String> tags, final String unit, final String metricType
+            final Map<String, String> tags, final Map<String, String> resources, final String unit,
```

Comment:
if another parameter gets added we should employ the "Introduce Parameter Object" refactoring as this number of parameters is unmanageable.